src/spec/expectations.cr

```diff
@@ -417,6 +417,26 @@ module Spec
   end
 
   module ObjectExtensions
+    # Validates an expectation and fails the example if it does not match.
+    #
+    # This overload will be called when doing:
```

Comment:
I'd avoid placing an example block inside a sentence.
```suggestion
    # This overload accepts a `be_a(Type)` expectation and makes sure, at compile-time, that the returned object is of type `Type`.
```

Reply:
Any reason for not mentioning the `be_a` method here? It would make the documentation more accessible for readers familiar with the DSL.

Reply:
Sorry, what do you mean? I changed the code example and `be_a` is used there.

Reply:
I meant mentioning it in the text.

But yeah, having it in the sample is probably fine.